consult.el

```diff
@@ -1312,5 +1317,40 @@ Prepend PREFIX in front of all items."
 
 (add-hook 'consult-preview-mode-hook #'consult--icomplete-preview-setup)
 
+(defun consult--flymake-candidates ()
+  "Return Flymake errors as alist."
+  (consult--forbid-minibuffer)
+  (let* ((diagnostics (flymake-diagnostics))
+	 (buffer-width (apply #'max (mapcar (lambda (x) (length (format "%s" (flymake-diagnostic-buffer x)))) diagnostics)))
+	 (category-width (apply #'max (mapcar (lambda (x) (length (symbol-name (flymake-diagnostic-type x)))) diagnostics)))
+         (fmt (format "%%-%ds %%-%ds %%s" buffer-width category-width)))
+    (mapcar
+     (lambda (diag)
+       (with-current-buffer (flymake--diag-buffer diag)
```

Comment:
Fix indentation using emacs please.

Reply:
I think this is done. I've converted tabs to spaces, and indented using indent-pp-sexp?